### DISCUSSION TITLE
Posting new Thunderbird support questions to a Matrix chat room

### DISCUSSION THREAD

**Discussion by jaller94** [windmill-labs/windmill#1751] [Show and tell]
I've built a small flow to post [Thunderbird Support questions](https://support.mozilla.org/en-US/questions/thunderbird) to a [Matrix](https://matrix.org/) chat room, which a support team can use to get notified and coordinate responses.

![matrix-room-preview](https://github.com/windmill-labs/windmill/assets/10872136/aeb85f01-0521-417a-9ba8-e9b3849379ba)

If you want to have a look at the flow, you can find the code here:
https://gitlab.com/jaller94/thunderbird-support-matrix-bot-windmill

## The flow

1. Fetch new questions from the [Thunderbird's public support forum](https://support.mozilla.org/en-US/questions/thunderbird).
2. Post new questions (link + initial question) in a Matrix room.

It does not:
* Allow to respond to questions within Matrix.
* Update edited or deleted questions.
* Use the End-to-End-Encryption of Matrix. They are public anyway.
* Post new questions as soon as they get posted. As far as I know the Thunderbird Support forum has no webhook support. Instead, this flow should be run on a schedule.

![flow](https://github.com/windmill-labs/windmill/assets/10872136/fb45573c-a13b-4bed-80df-ae4a42f3f920)

**Comment by rubenfiszel** [comment]
This is so cool! Do you think it would make sense to publish that as well on the hub and backlinking this post in the description ?
